packages/myst-to-jats/src/index.ts

```diff
@@ -129,18 +138,26 @@ function mathToMml(math?: string, inline?: boolean) {
   if (!mathElement) return;
   if (inline) mathElement.attributes = { ...mathElement.attributes, display: 'inline' };
   delete mathElement.attributes?.xmlns;
-  function addMmlAndRemoveAnnotation(el?: Element) {
-    if (el?.name) el.name = `mml:${el.name}`;
-    if (!el?.elements) return;
-    el.elements = el.elements.filter((child: Element) => child.name !== 'annotation');
-    el.elements.forEach((child: Element) => {
-      addMmlAndRemoveAnnotation(child);
-    });
-  }
   addMmlAndRemoveAnnotation(mathElement);
+  // Remove the wrapping `<mml:semantics><mml:mrow>` if it is the only element
+  if (mathElement?.elements?.length === 1 && mathElement.elements[0].name === 'mml:semantics') {
+    mathElement.elements = mathElement.elements[0].elements;
+  }
+  if (mathElement?.elements?.length === 1 && mathElement.elements[0].name === 'mml:mrow') {
+    mathElement.elements = mathElement.elements[0].elements;
+  }
   return mathElement;
 }
 
+function cleanLatex(value?: string): string | undefined {
```

Comment:
Maybe add a docstring for what is cleaned?

```typescript
/**
 * Remove comments and consolidate to one line
 */
```

Reply:
```suggestion
/**
 * Remove comments and consolidate to one line
 */
function cleanLatex(value?: string): string | undefined {
```